onnxruntime/contrib_ops/cuda/moe/ft_moe/common.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2020-2021, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <iostream>
+#include <cuda_runtime.h>
+#include <cuda_fp16.h>
+#include <cublas_v2.h>
+#include <cublasLt.h>
+#include <stdexcept>
+#include <map>
+#include "stdio.h"
+#include <fstream>
+
+namespace fastertransformer {
+
+static const char* _cudaGetErrorEnum(cublasStatus_t error) {
```

Comment:
Can we use https://github.com/microsoft/onnxruntime/blob/c250540722a8028250de1d572365ec73ef6ecedf/onnxruntime/core/providers/cuda/cuda_common.cc#L30